.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file needs to be in order to run correctly prettier conflict tests due to these changes: https://github.com/prettier/eslint-config-prettier#cli-helper-tool

Supplying a custom config to the command doesn't seem to work and I couldn't find any extended documentation about this.